app/models/user.rb

```diff
@@ -11,6 +11,8 @@ class User < ApplicationRecord
     if: :address_changed?
 
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Style/StringLiterals
**Severity**: INFO
**File**: app/models/user.rb L12

Prefer single-quoted strings when you don&apos;t need string interpolation or special symbols.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*